pymlx/featurizer.py

```diff
@@ -7,13 +7,40 @@
 from .features_handler import *
 
 
+def listable(t):
+    return Union[t, List[t]]
+
+
 class Featurizer:
-    def __init__(self, handlers: List[FeaturesHandler], sparse=False):
-        assert isinstance(handlers, list)
+    def __init__(self,
+                 handlers: listable(FeaturesHandler) = None,
+                 no_op: listable(str) = None,
+                 categorical: listable(str) = None,
+                 one_hot: listable(str) = None,
+                 binning: listable(str) = None, sparse=False):
+
+        _handlers = []
+
+        if no_op is not None:
+            _handlers.append(NoHandler(no_op))
```

Comment:
If `no_op` is a List, you need to use `extend`, not `append`, right?

Reply:
no_op is a list but I put it in `NoHandler`. `_handlers` is a list of `FeatureHandler`.